### PR TITLE
Create ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-58182](https://cumulocity.atlassian.net/browse/MTM-58182)
Original PR: [7507](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7507)

[MTM-58182]: https://cumulocity.atlassian.net/browse/MTM-58182?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: emit move event only when dragstart event emits (#7376) (#7507)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md]: The Claude AI suggested to use the following title:
```suggestion
title: The map widget's center coordinate inputs in the configuration should not trigger move events when modified
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58182
+version: 1021.0.8
+---
+emit move event only when dragstart event emits (#7376) (#7507)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md]: The Claude AI suggested to use the following body:
```suggestion
The map widget's center coordinate inputs in the configuration should not trigger move events when modified. This issue was leading to a situation where a move event was emitted, which resulted in updates to the center coordinate inputs.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md (hunk 2 above)
Comment (rahulc8y) [reply content/change-logs/application-enablement/ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md]: As a user, it's not clear if the above changelog belongs to an issue regarding which component or feature of the application (in this case: map widget)

**Review by DawidSAG — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58182
+version: 1021.0.8
+---
+The map widget's center coordinate inputs in the configuration should not trigger move events when modified. This issue was leading to a situation where a move event was emitted, which resulted in updates to the center coordinate inputs.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md]: I do not really get what this change is about from a user perspective. What is the fix?

Comment (BeateRixen) [reply content/change-logs/application-enablement/ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md]: @DawidSAG @janhommes @rahultsag Please check this PR which needs rephrasing

Comment (janhommes) [reply content/change-logs/application-enablement/ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md]: @BeateRixen I updated it, can you check?

File: content/change-logs/application-enablement/ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58182
+version: 1021.0.8
+---
+The map widget's center coordinate inputs in the configuration had issues setting the right coordinates by user input. The fix ensures that the user can set the right coordinates also via text input and not only by dragging the map.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md]:
```suggestion
In the configuration of the "Map widget" the center coordinates provided by the user via text input were not set correctly. This change ensures that the user can set the right coordinates also via text input and not only by dragging the map.
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: The map widget's center coordinate inputs in the configuration should work consistently
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md]:
```suggestion
title: Map widget's center coordinate inputs work consistently
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md (hunk 3 above)
Comment (BeateRixen) [reply content/change-logs/application-enablement/ui-c8y-1021-0-8-emit-move-event-only-when-dragstart-event-emits-7376-7507.md]: @janhommes Please check ones more

**Review by janhommes — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**